templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -671,6 +679,11 @@ Resources:
         PublicSubnet1CIDR: !Ref 'PublicSubnet1CIDR'
         PublicSubnet2CIDR: !Ref 'PublicSubnet2CIDR'
         VPCCIDR: !Ref 'VPCCIDR'
+        QSS3BucketName: !Ref 'QSS3BucketName'
```

Comment:
What's the purpose of this addition?

Reply:
`quickstart-vpc-for-atlassian-services.yaml ` uses these parameters. When they are not passed, the template looks for the wrong bucket and fails.